packages/react-positioning/src/createArrowStyles.ts

```diff
@@ -67,15 +77,15 @@ export function createArrowStyles(size?: number): MakeStylesStyleRule<Theme> {
       },
 
       ':global([data-popper-placement^="right"])': {
-        left: 0,
+        [dir === 'ltr' ? 'left' : 'right']: 0,
```

Comment:
seems like there is enough logic in here now to do a few tests with the makestyles serializer ?